src/database/migrations/20241212203721-add-role-permissions-for-reports.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+require('module-alias/register')
+const userRequests = require('@requests/user')
+require('dotenv').config()
+const common = require('@constants/common')
+const Permissions = require('@database/models/index').Permission
+
+const getPermissionId = async (module, request_type, api_path) => {
+	try {
+		const permission = await Permissions.findOne({
+			where: { module, request_type, api_path },
+		})
+		if (!permission) {
+			throw error
+		}
+		return permission.id
+	} catch (error) {
+		throw error
+	}
+}
+
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		try {
+			const rolePermissionsData = [
+				{
+					role_title: common.ORG_ADMIN_ROLE,
+					permission_id: await getPermissionId('reports', ['GET'], '/mentoring/v1/reports/filterList'),
```

Comment:
@sumanvpacewisdom these api's can be accessed by the mentee,mentor, org admin and session manager. 
Provide permission for those roles as well.